stack.yaml

```diff
@@ -214,10 +223,6 @@ extra-deps:
 
 # Serokell-specific utilities not on stackage
 - fmt-0.6
-# We *must* use 6.3.6 at least, otherwise `int` is synonymous with `build` and
-# will result in an infinite loop -> space leak; however we can't yet because
-# CI hasn't run cabal update yet.
-# - formatting-6.3.6
```

Comment:
I thought this issue with formating had been fixed.

Reply:
Right, formatting was just in the stack.yaml twice. The other one is the one you linked to:

https://github.com/input-output-hk/cardano-sl/blob/d91b66c41c753abf2c93fd9cf8ffb976bde5669b/stack.yaml#L247-L249

So this is just dead code removal.